pages/event-types/[type].tsx

```diff
@@ -1285,16 +1276,14 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
 
   // backwards compat
   if (eventType.users.length === 0) {
-    eventType.users.push(
-      await prisma.user.findUnique({
-        where: {
-          id: session.user.id,
-        },
-        select: {
-          username: true,
-        },
-      })
-    );
+    const fallbackUser = await prisma.user.findUnique({
+      where: {
+        id: session.user.id,
+      },
+      select: userSelect,
+    });
+    if (!fallbackUser) throw Error("The event type doesn't have user and no fallback user was found");
```

Comment:
The situation will occur when there is a team event with zero attendees. See above suggestion to resolve.

Reply:
Nice catch!